<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KouKou</class>
 <widget class="QMainWindow" name="KouKou">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>587</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>KouKou</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/2.jpg</normaloff>:/Background_res/2.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/3.jpg</normaloff>:/Background_res/3.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/5.jpg</normaloff>:/Background_res/5.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>神奇魔法：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/0.jpg</normaloff>:/Background_res/0.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/1.jpg</normaloff>:/Background_res/1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>561</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/4.jpg</normaloff>:/Background_res/4.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/6.jpg</normaloff>:/Background_res/6.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_9">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/7.jpg</normaloff>:/Background_res/7.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/8.jpg</normaloff>:/Background_res/8.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_11">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Background_res/9.jpg</normaloff>:/Background_res/9.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择背景：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>530</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>自定义</string>
    </property>
   </widget>
   <zorder>toolButton_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>toolButton_5</zorder>
   <zorder>toolButton_7</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>toolButton_2</zorder>
   <zorder>toolButton_3</zorder>
   <zorder>label</zorder>
   <zorder>toolButton_6</zorder>
   <zorder>toolButton_8</zorder>
   <zorder>toolButton_9</zorder>
   <zorder>toolButton_10</zorder>
   <zorder>toolButton_11</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
